doc/cookbook/error_handler.rst

```diff
@@ -36,3 +36,18 @@ the ``$debug`` argument::
     use Symfony\Component\Debug\ExceptionHandler;
 
     ExceptionHandler::register(false);
+
+
+.. note::
+
+    Important caveat when using Silex in a command-line interface:
+    The ``ExceptionHandler`` should not be enabled as it would convert an error
+    to HTML output and return a non-zero exit code.
+
+    .. code-block:: php
```

Comment:
Should be removed in favor of replacing the `.` at the end of the previous sentence by `::`